src/libs/tokens/tokenHelperFn.ts

```diff
@@ -15,10 +14,14 @@ const buildIpfsUri = (ipfsHash: string) => `https://ipfs.io/ipfs/${ipfsHash}`;
 export const fetchTokenLists = async () => {
   const res = await Promise.all(
     config.tokenLists.map(async (uri) => {
-      const res = await axios.get<TokenList>(uri, { timeout: 10000 });
+      const signal = AbortSignal.timeout(10000);
+      const res: TokenList = await fetch(uri, {
+        method: 'GET',
+        signal,
+      }).then((response) => response.json());
```

Comment:
You don't need the method: "GET", this is the default.
Also you need to check for error : 
```suggestion
      const response = await fetch(uri, { signal });
      if (!response.ok) throw new Error(....);
      const result: TokenList  = await response.json());
```